R/pkg/R/DataFrame.R

```diff
@@ -411,7 +411,7 @@ setMethod("coltypes",
 #'
 #' Set the column types of a SparkDataFrame.
 #'
-#' @param x A SparkDataFrame
+#' @param y A SparkDataFrame whose column types to be set
```

Comment:
to avoid duplicate variables, now it has 2 'x' in 'ARGUMENTS' section, or is there a better way to do it?

http://spark.apache.org/docs/2.0.0/api/R/coltypes.html

https://cloud.githubusercontent.com/assets/3925641/17386808/effb98ce-59a2-11e6-9657-d477d258a80c.png